Innowise.Clinic.Auth.Jwt/TokenGenerator.cs

```diff
@@ -0,0 +1,73 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Innowise.Clinic.Auth.Jwt.Interfaces;
+using Innowise.Clinic.Auth.Persistence;
+using Innowise.Clinic.Auth.Persistence.Models;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Innowise.Clinic.Auth.Jwt;
+
+public class TokenGenerator : ITokenGenerator
```

Comment:
Generators are associated with static classes so rename it to smth like `TokenService`.